src/runner.ts

```diff
@@ -487,11 +498,45 @@ export class ActionRunner {
         },
         rule.minRank,
       );
-    } else {
-      this.logger.info("Rule requirements fulfilled");
-      // If we don't have any missing reviews, we return no error
-      return null;
+      // Then we verify if we need to have a minimum score
+    } else if (rule.minScore && scores) {
+      // We get all the fellows with their ranks and convert them to their score
+      const fellows: [string, number][] = (await this.polkadotApi.listFellows()).map(([handle, rank]) => [
```

Comment:
perhaps worth checking if fellows are returned, otherwise it will fulfill the rules positively if polkadotApi returns nothing

Reply:
Regarding this we have two checks:
The first one is [here](https://github.com/paritytech/review-bot/blob/4bd69c417e8acba07e03c8b831e5fa1159739154/src/runner.ts#L461) to ensure that we always have the amount of requested reviewers.

The second one is [here](https://github.com/paritytech/review-bot/blob/4bd69c417e8acba07e03c8b831e5fa1159739154/src/runner.ts#L511) to ensure that, if we use the scores, all the fellows available have enough score to fulfill the requirements

Reply:
right, looks like if `await this.polkadotApi.listFellows()` ends up with empty list, there will be "Minimum score" exception. Should work though 👌